data/en/census_household.json

```diff
@@ -3828,7 +3828,7 @@
                         },
                         {
                             "goto": {
-                                "id": "visitor-completed"
+                                "block": "visitor-completed"
                             }
                         }
                     ]
```

Comment:
Do we need to update welsh versions?

Reply:
Yes

Reply:
pipenv run ./scripts/test_schemas.sh

Reply:
👍

Reply:
i can't see any changes to cy files

Reply:
Given that test_schemas doesnt actually test the welsh schemas we should add this here.
https://github.com/ONSdigital/eq-survey-runner/blob/92282d2cb76a3ce53b8bb0dbf6b9ca76e5d8047f/scripts/build.sh#L19

Reply:
was meant to say `tralate_schemas`